libs/components/core/src/lib/modules/affix/dom-utils.ts

```diff
@@ -45,6 +45,13 @@ export function getElementOffset(
   };
 }
 
+export function getViewportOffset(): { top: number; left: number } {
```

Comment:
Let's rename this to `getVisualViewportOffset` to make it more clear which viewport we're referencing.

Reply:
I moved away from this solution.